modules/ols-0-3-6-known-issues.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is used in the following assemblies:
+// * lightspeed-docs-main/release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-0-3-6-known-issues_{context}"]
+= Known issues
+
+The following issues are identified with {ols-official} 0.3.6:
+
+* For {ocp-product-title} 4.17 and later, the {ols-short} icon disappears when you click *Create Namespace* or *Create Project* from one of the following locations in the {ocp-product-title} web console:
+
+** *Administration -> Namespaces* 
+** *Home -> Projects -> Create Project*
+** The *Project* drop-down menu located at the top of most pages
```

Comment:
I _think_ to get the workaround line to properly indent at the right level, you would want to wrap this in something like a -- block:
```suggestion
--
** *Administration -> Namespaces* 
** *Home -> Projects -> Create Project*
** The *Project* drop-down menu located at the top of most pages
--
```

But I don't think it's worth delaying the merging over, feel free to try that in a follow up PR